static/app/components/alerts/notificationBar.tsx

```diff
@@ -21,3 +21,5 @@ export const NotificationBar = styled('div')`
     color: ${p => p.theme.alert.info.iconColor};
   }
 `;
+
+export default NotificationBar;
```

Comment:
you're exporting it both ways now. Do we need the default export? I'm fine with just a named export